packages/react/src/Grid/Grid.tsx

```diff
@@ -29,17 +32,28 @@ type GridPaddingTopAndBottomProps = {
 }
 
 export type GridProps = {
+  /** The HTML tag to use. */
+  as?: GridTag
   /** The amount of space between rows. */
   gapVertical?: 'none' | 'small' | 'large'
 } & (GridPaddingVerticalProp | GridPaddingTopAndBottomProps) &
   PropsWithChildren<HTMLAttributes<HTMLDivElement>>
 
 const GridRoot = forwardRef(
   (
-    { children, className, gapVertical, paddingBottom, paddingTop, paddingVertical, ...restProps }: GridProps,
-    ref: ForwardedRef<HTMLDivElement>,
+    {
+      as: Tag = 'div',
+      children,
+      className,
+      gapVertical,
+      paddingBottom,
+      paddingTop,
+      paddingVertical,
+      ...restProps
+    }: GridProps,
+    ref: any,
```

Comment:
Can't we use `ForwardedRef<HTMLElement>` here?

Reply:
No, because it can be a `div` and `HTMLDivElement` requires the `align` attribute.

Reply:
Maybe `ForwardedRef<any>`? So we at least check if the ref is of type `MutableRefObject`?

Reply:
Sure!